easybuild/easyconfigs/g/GROMACS/GROMACS-4.6.5-intel-2015a.eb

```diff
@@ -0,0 +1,23 @@
+name = 'GROMACS'
+version = '4.6.5'
```

Comment:
@molden: are you aware there's a 4.6.7 too? just wondering, I'm happy to include this one too